### DISCUSSION TITLE
Probing Progress not being displayed in Octoprint Print Notifications section

### DISCUSSION THREAD

**Discussion by jwolach** [Jyers/Marlin#826] [Q&A] [answered]
I recently upgraded my Ender 3 v2 firmware to your 1.3.2 with HS BLTouch.  The previous firmware I was running was from Smith3D.  When I was running the Smith3D firmware with my Octoprint, I was able to see the Probing progress (Probing 1/25, 2/25..etc)  in the Printer Notifications section of Octoprint.  With the Jyers 1.3.2 firmware, the Probing progress does not display.

Is there somewhere in the source code that I can turn that on?

Thank you very much for an AWESOME firmware!

**Comment by Jyers** [comment, marked as answer]
Interesting, I don't know why this would be handled any differently. It might be fixed in the next release due to the big code rebase. I'll take a look

**Reply by jwolach** [reply]
Hi Jyers,

I just noticed on my printer's LCD that the Probing progress is displayed (see photo).  However, it's not showing up in my Octoprint.  Very strange!!

![IMG_1225](https://user-images.githubusercontent.com/7068422/118867025-a233f100-b8b0-11eb-837a-da822c5c2d86.png)

**Comment by jwolach** [comment]
I would appreciate it.  Attached is a photo of my Octoprint Printer Notification section when my printer was running the Smith3D Marlin 2.0.x firmware.

![IMG_1216](https://user-images.githubusercontent.com/7068422/118539425-6ae10b00-b71d-11eb-8508-1f1ada3fab1a.png)
